docs/changelog.rst

```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :bug:`-` Velodrome and Aerodrome LP pools will now refresh properly.
```

Comment:
can you put it with the other bugfixes? To not have them mixed features and bugs